sql/src/test/java/io/crate/analyze/SetAnalyzerTest.java

```diff
@@ -23,18 +23,18 @@
 package io.crate.analyze;
 
 import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.LinkedListMultimap;
```

Comment:
The logic that you have moved to plan classes for `UpdateSettingsPlan` and `SetLicensePlan` is not tested. You probably have to extend the `SetAnalyzerTest#analyze` method to cover it. Please, see the other analyzer tests for example.

Reply:
I moved the test to https://github.com/crate/crate/blob/dd4ffef96232c8c2e35ca6f231630f90f0f72b71/sql/src/test/java/io/crate/planner/node/ddl/UpdateSettingsPlanTest.java#L131-L130 , is that not the way to go ?

Reply:
ah i see, missed that for some reason, sorry